code/scripting/ade_api.h

```diff
@@ -276,7 +303,31 @@ class ade_virtvar : public ade_lib_handle {
  */
 #define ADE_VIRTVAR(name, parent, args, desc, ret_type, ret_desc)                                                      \
 	static int parent##_##name##_f(lua_State* L);                                                                      \
-	::scripting::ade_virtvar parent##_##name(#name, parent##_##name##_f, parent, args, desc, ret_type, ret_desc);      \
+	::scripting::ade_virtvar parent##_##name(#name, parent##_##name##_f, parent, args, desc, ret_type, ret_desc,       \
+	                                         ::gameversion::version(), nullptr);                                       \
+	static int parent##_##name##_f(lua_State* L)
+
+/**
+ * @brief Declare an API variable
```

Comment:
Should probably change this to say "Declare a **deprecated** API variable".

Reply:
Fixed and also fixed the various clang-tidy issues.